pom.xml

```diff
@@ -483,6 +483,22 @@ THE SOFTWARE.
           <rerunFailingTestsCount>4</rerunFailingTestsCount>
         </configuration>
       </plugin>
+      <plugin>
+        <artifactId>maven-javadoc-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>run-javadoc</id>
```

Comment:
I see no need in it. Javadoc check is pretty fast. And it is no safe to do it, because I may need to release with -skipTests on Windows (really bad practice, yeah)